src/rdcrc32.h

```diff
@@ -70,7 +70,7 @@ rd_crc32_t rd_crc32_reflect(rd_crc32_t data, size_t data_len);
  *****************************************************************************/
 static __inline rd_crc32_t rd_crc32_init(void)
 {
-#if WITH_ZLIB
+#if WITH_ZLIB && !defined(_MSC_VER)
```

Comment:
I dont really understand what you are trying to achieve here.
Are you trying to disable the use of zlib on win32? If so you should remove WITH_ZLIB from src/win32_config.h, but that seems questionable.